backend/tracim_backend/lib/rq/worker.py

```diff
@@ -69,7 +69,7 @@ def worker_context() -> typing.Generator[TracimContext, None, None]:
         yield context
         transaction.commit()
     except Exception:
-        transaction.rollback()
+        transaction.abort()
```

Comment:
What the difference between abort and rollback ?

Reply:
`transaction` uses `abort()` as I found out (`rollback()` doesn't exist).